pom.xml

```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>org.eclipse.microprofile</groupId>
         <artifactId>microprofile-parent</artifactId>
-        <version>2.4</version>
+        <version>3.4</version>
```

Comment:
This cannot be done in this way.

First, don't do dependency upgrades in the feature PRs.
Second, this would move minimum JDK from 8 to 11 which can only be done in major upgrade of the specification.

Reply:
+1, I will remove that commit from this PR